### ISSUE TITLE
[FEAT] Some suggestions for this software

### ISSUE THREAD

**Issue by SMFDrummer** [eythaann/Seelen-UI#199]
Since the software was released, the feedback from all users who have installed and used it on any platform is mostly negative, such as: if it is set arbitrarily, the small triangle on the toolbar cannot be clicked and keeps flashing, it should trigger a bug and loop on its own. It is already version 2.0.3 and the most basic stability of the software has not been resolved. The frequency of bug triggers is even more severe than MyDockFinder. If developers still cannot effectively solve various bugs that even affect normal use, it is recommended to release all projects to pre release and indicate the serious problems that may be caused by user use. You can also write down all the effective solutions for the issues raised in a markdown file for users to refer to. If it still cannot effectively solve various bug problems and blindly innovate, then I suggest developers slow down their development speed. If all users are attracted by your appearance and uninstall your software due to serious bugs in the program, how many people will pay attention to and use your software in the end?

**Comment by eythaann** [comment]
yep, you have a big point on this, but, just wait. I'm currently alone on this project so I can't support all the stuff that a big project needs only by myself. Also the project is free so I'm not getting funds for the app to start getting more people into the project.

Now, growth just happens I wasn't planning on promoting the app yet, just blogs start posting about the app.

btw, this is normal when fasting growing, if nothing is on fire, you're not moving fast enough.
